migration.go

```diff
@@ -53,13 +55,16 @@ func LoadMigration(absFilename string) (m Migration, err error) {
 // ToYAML returns the contents of a YAML encoded fixture.
 func (m Migration) ToYAML() ([]byte, error) {
 	out := new(bytes.Buffer)
```

Comment:
Why not just remove the comment if you are happy with new implementation?

Reply:
will do